homeassistant/components/homewizard/strings.json

```diff
@@ -66,19 +66,19 @@
         "name": "Total energy export tariff {tariff}"
       },
       "active_power_w": {
-        "name": "Active power"
+        "name": "Power"
       },
```

Comment:
I am guessing this has device class power -> In that case, remove the translation key entirely as it will use the device class translation -> less duplicate translation keys :)

Reply:
O nice!